README.md

```diff
@@ -1,45 +1,54 @@
-# **hipBLAS**
+# hipBLAS
 
-**hipBLAS** is a Basic Linear Algebra Subprograms (**BLAS**) marshalling library, with multiple supported backends.  It sits between the application and a 'worker' BLAS library, marshalling inputs into the backend library and marshalling results back to the application.  **hipBLAS** exports an interface that does not require the client to change, regardless of the chosen backend.  Currently, **hipBLAS** supports **rocBLAS** and **cuBLAS** as backends.
+hipBLAS is a Basic Linear Algebra Subprograms (BLAS) marshalling library with multiple supported
+backends. It sits between your application and a 'worker' BLAS library, where it marshals inputs to the
+backend library and marshals results to your application.  hipBLAS exports an interface that doesn't
+require the client to change, regardless of the chosen backend. Currently, hipBLAS supports rocBLAS
+and cuBLAS backends.
 
-## Documentation
-
-For a detailed description of the **hipBLAS** library, its implemented routines, the installation process and user guide, see the [**hipBLAS** Documentation](https://hipblas.readthedocs.io/en/latest/).
+To use hipBLAS, you must first install rocBLAS and rocSOLVER or cuBLAS.
```

Comment:
rocSPARSE is a dependency of rocSOLVER as well, not sure if that should be included here.

Reply:
Sure, I'll add it in